schemas/context/segmentmembership.schema.json

```diff
@@ -29,6 +29,66 @@
           "format": "date-time",
           "description": "The timestamp when the assertion of segment membership was made."
         },
+        "xdm:payload" : {
+          "title" : "Payload",
+          "description": "Values that are directly related with the segment realization. This payload exists with the same validUntil as the segment realization.",
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "xdm:payloadNumberValue": {
+                  "title": "Value",
+                  "type": "number",
+                  "description": "The number"
+                },
+                "xdm:payloadType": {
+                  "title": "Payload Type",
+                  "type": "string",
+                  "const": "number",
+                  "description": "The type of payload."
+                }
+              },
+              "required": ["xdm:payloadNumberValue", "xdm:payloadType"]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "xdm:payloadPropensityValue": {
```

Comment:
The thinking is that propensity can live outside of payload and belongs more with the core of a what segment is.

Reply:
We revisited this with @kstreeter and decided to bring back propensity here.